src/services/api/sites.js

```diff
@@ -3,7 +3,8 @@ import { SITES } from 'common/constants'
 
 const site = {
   get: async ({ sitename }) => apiBase().get(`${SITES}/${sitename}/`).json(),
-  getAll: async () => apiBase().get(SITES).json(),
+  getAll: async (page = 1, pageSize = 100) =>
```

Comment:
In the interest of being explicit, and consistent with the other apis and dataHooks, please destructure the props for getAll and useSites.
e.g. `getAll: async ({ page = 1, pageSize = 100 })` and when called `api.sites.getAll({ pageSize })`